roles/test_report_send/tasks/reporting/validations.splunk.yml

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file for redhatci.ocp.test_report_send
+
+- name: Validate collector connectivity var trs_collector_url is not empty
+  ansible.builtin.assert:
+    that:
+      - trs_collector_url | default("") | length > 0
+    fail_msg:
+      - "Variable trs_collector_url is either undefined or empty"
+  tags:
+    - inproc
+    - validation
```

Comment:
`trs_collector_url` must be defined per args spec.  Validation of length happens already in main.yml

```suggestion
```